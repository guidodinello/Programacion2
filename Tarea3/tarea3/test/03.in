# prueba de imprimirBinario
imprimirBinario
insertarEnBinario (5,5.5)
imprimirBinario
insertarEnBinario (3,3.3)
insertarEnBinario (6,6.6)
imprimirBinario
insertarEnBinario (2,2.2)
insertarEnBinario (7,7.7)
insertarEnBinario (9,9.9)
insertarEnBinario (8,8.8)
imprimirBinario
Fin