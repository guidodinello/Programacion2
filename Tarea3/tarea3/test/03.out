1>#  prueba de imprimirBinario.
2>
3>Insertado.
4>
(5,5.50)
5>Insertado.
6>Insertado.
7>
-(6,6.60)
(5,5.50)
-(3,3.30)
8>Insertado.
9>Insertado.
10>Insertado.
11>Insertado.
12>
---(9,9.90)
----(8,8.80)
--(7,7.70)
-(6,6.60)
(5,5.50)
-(3,3.30)
--(2,2.20)
13>Fin.
